src/cr/cube/noa/smoothing.py

```diff
@@ -146,4 +154,4 @@ def _smoothed_values(self):
     @lazyproperty
     def _window(self):
         """int, the window parameter specified in the measure expression"""
-        return self._measure_expression.get("window", 3)
+        return self._measure_expression.get("window", 2)
```

Comment:
It seems at first glance that you don't have a test that tests a default value (if it's not provided in the expression).